commune_app/models.py

```diff
@@ -1,3 +1,8 @@
-# from django.db import models
+from django.db import models
 
```

Comment:
add another blank newline

Reply:
ye i did that first time then fixed it with the second commit